llvm/lib/Transforms/InstCombine/InstCombineShifts.cpp

```diff
@@ -994,6 +994,18 @@ static bool setShiftFlags(BinaryOperator &I, const SimplifyQuery &Q) {
       I.setIsExact();
       return true;
     }
+    //Fix #131444
```

Comment:
Don't reference the issue, instead add a compact comment on what the transform does. Like `// Infer exact on shift by cttz`.